docs/ops/sequence/CTCGreedyDecoderSeqLen_6.md

```diff
@@ -0,0 +1,78 @@
+## CTCGreedyDecoderSeqLen <a name="CTCGreedyDecoderSeqLen"></a> {#openvino_docs_ops_sequence_CTCGreedyDecoderSeqLen_6}
+
+**Versioned name**: *CTCGreedyDecoderSeqLen-6*
+
+**Category**: Sequence processing
+
+**Short description**: *CTCGreedyDecoderSeqLen* performs greedy decoding on the logits given in input (best path) uses 1D tensor with sequence lengths.
+
+**Detailed description**:
+
+This operation is similar [Reference](https://www.tensorflow.org/api_docs/python/tf/nn/ctc_greedy_decoder)
+
+Given an input sequence \f$X\f$ of length \f$T\f$, *CTCGreedyDecoderSeqLen* assumes the probability of a length \f$T\f$ character sequence \f$C\f$ is given by
+\f[
+p(C|X) = \prod_{t=1}^{T} p(c_{t}|X)
+\f]
+
+Sequences in the batch can have different length. The lengths of sequences are coded in the second input integer tensor `sequence_length`.
+
+The main difference between CTCGreedyDecoder and CTCGreedyDecoderSeqLen is in second input. CTCGreedyDecoder uses 2D input floating point tensor with sequence masks for each sequence in the batch while CTCGreedyDecoderSeqLen uses 1D integer tensor with sequence lengths.
+
+**Attributes**
+
+* *merge_repeated*
+
+  * **Description**: *merge_repeated* is a flag for merging repeated labels during the CTC calculation. If value is false the squense A B B * B * B  (where '*' is the blank label) will look like ABB * B * B. But if we set it to true, the squensy will be AB * B * B. A B B will be merged repeted to A B.
```

Comment:
1. type squense
2. please remove spaces in examples: `ABB*B*B`, etc.
3. also the resulted sequence is incorrect. If `merge_repeated = false`, then `ABB*B*B` -> `ABBBB`
If `merge_repeated = true`, then `ABB*B*B` -> `ABBB`

Reply:
Please check